src/Aspire.Hosting/ParameterResourceBuilderExtensions.cs

```diff
@@ -216,7 +216,6 @@ internal static IResourceBuilder<T> AddParameter<T>(this IDistributedApplication
         {
             ResourceType = KnownResourceTypes.Parameter,
             Properties = [
-                new("parameter.secret", resource.Secret.ToString()),
```

Comment:
This is removed but I'm not seeing where it is injected elsewhere, are we just relying on parameter value now?